contracts/RiskFund/ReserveHelpers.sol

```diff
@@ -102,10 +126,7 @@ contract ReserveHelpers is Ownable2StepUpgradeable {
         require(ComptrollerInterface(comptroller).isComptroller(), "ReserveHelpers: Comptroller address invalid");
         address poolRegistry_ = poolRegistry;
         require(poolRegistry_ != address(0), "ReserveHelpers: Pool Registry address is not set");
-        require(
-            PoolRegistryInterface(poolRegistry_).getVTokenForAsset(comptroller, asset) != address(0),
-            "ReserveHelpers: The pool doesn't support the asset"
-        );
+        require(ensureAssetListed(comptroller, asset), "ReserveHelpers: The pool doesn't support the asset");
```

Comment:
```suggestion
        require(ensureAssetListed(comptroller, asset), "ReserveHelpers: Asset not supported by any pool");
```

Reply:
I think we can use custom errors instead of require

Reply:
I would reduce the number of changes, due to the temporary nature of this contract